File: UI/eturistickaagencija_admin/lib/models/rezervacija.dart
import 'package:json_annotation/json_annotation.dart';

part 'rezervacija.g.dart';

@JsonSerializable()
class Rezervacija{
int? id;
double? cijena;
int? hotelId;
bool? otkazana;
DateTime? datumRezervacije;
int? korisnikId;

Rezervacija(this.id,this.cijena,this.hotelId,this.otkazana,this.datumRezervacije,this.korisnikId);

factory Rezervacija.fromJson(Map<String,dynamic>json)=>_$RezervacijaFromJson(json);
Map<String,dynamic>toJson()=>_$RezervacijaToJson(this);
}

File: eTuristickaAgencija.Service/Database/Rezervacija.cs
using System;
using System.Collections.Generic;

namespace eTuristickaAgencija.Service.Database;

public partial class Rezervacija
{
    public int Id { get; set; }

    public decimal Cijena { get; set; }

    public int? HotelId { get; set; }

    public bool? Otkazana { get; set; }

    public DateTime DatumRezervacije { get; set; }

    public int? KorisnikId { get; set; }

    public virtual Hotel? Hotel { get; set; }

    public virtual Korisnik? Korisnik { get; set; }
}
